Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -1,417 +1,418 @@
+defensive implement in his or her off-hand, that item cannot be used and 
+it provides no bonuses for one round.
+
+==============================
+UNARMED
+==============================
+
+== Unarmed Strike ==
+
+Damage: 20 + STR mod + DEX mod, minimum 10
+Range: Adjacent (1m)
+Accuracy Modifier: 0
+
+==============================
+MELEE WEAPONS
+==============================
+
+===== LEVEL 1 REQUIREMENT WEAPONS =====
+
+== Eliazar Plasma Torch $240 ==
+
+Damage: 30 + 2x Plasma Burn
+Size: Light
+Range: Adjacent (1m)
+Accuracy Modifier: +1
+STR Requirement:   3
+
+	Plasma. Can quickly repair vehicles with welding materials, seal/cut through 
+moderate doors, and make temporary cover. -40% stealth for being quiet on stealth attacks.
+
+== Gurber G1 Combat Knife $315 ==
+
+Damage: 40 + STR + 2*DEX + 1xBleed
+Size: Light
+Range: Adjacent (1m)
+Accuracy Modifier: +1
+DEX Requirement: 3
+
+== Gurber GS101 Combat Sword $680 ==
+
+Damage: 50 + 2*STR + 2*DEX + 1xBleed
+Size: One-handed
+Range: 2m
+Accuracy Modifier: +2
+DEX Requirement: 4
+
+===== LEVEL 5 REQUIREMENT WEAPONS =====
+
+== Guri Grabber Knife $470 ==
+
+Damage: 30 + STR + 2*DEX + 1xBleed
+Size: Light
+Range: Adjacent (1m)
+Accuracy Modifier: +1
+DEX Requirement: 5
+
+	You can grapple after any successful basic melee attack. +10% Skill on 
+grappling checks. Whenever you roll a successful grappling check, you may deal 
+[3*STR or 3*DEX] damage to your opponent.
+
+== Guri Carbon Whip $560 ==
+
+Damage: 30 + 3*STR + 2*DEX
+Size: Exotic - One-handed
+Range: 3m
+Accuracy Modifier: +1
+DEX Requirement: 6
+
+	Can grapple limbs from range (DEX based grapple) and force movement 
+(1m for every 5 [DEX or STR]). Can make Disarm attempts at range. Grapple, 
+Shove, and Disarm attempts with the whip costs Actions as normal. This weapon
+has a Guard modifier of 0.
+
+== SPECO SP10 VAE Combat Knife $640 ==
+
+Damage: 45 + STR + 2*DEX + 1xBleed
+Size: Light
+Range:  Adjacent (1m)
+Accuracy Modifier: +1
+DEX Requirement: 4
+
+	Voltage Aligned Edge; ignores standard armor
+
+== SPECO SPC50 VAE Circular Saw $695 ==
+
+Damage: 55 + STR + 1xBleed
+Size: Two-handed
+Range: Adjacent (1m)
+Accuracy Modifier: +1
+STR Requirement: 4
+
+	Voltage Aligned Edge; ignores standard armor; can easily cut through most 
+materials. -50 on stealth rolls for being quiet during stealth attacks with 
+this weapon.
+
+== SPECO SP30 VAE Combat Sword $1245 ==
+
+Damage: 55 + 2*STR + 2*DEX + 1xBleed
+Size: One-handed
+Range: 2m
+Accuracy Modifier: +2
+DEX Requirement: 6
+
+	Voltage Aligned Edge; ignores standard armor
+
+===== LEVEL 10 REQUIREMENT WEAPONS =====
+
+== Eliazar E500 Assisted-Sledgehammer $811 ==
+
+Damage: 50 + 6*STR
+Size: Two-handed
+Range: 2m
+Accuracy Modifier: +0
+STR Requirement: 6
+
+	Considered concussive (explosive) damage. Hits armor points 
+before physical damage if the target's armor has defense against explosive (i.e. 
+concussive) damage. If armor roll is greater than 50%, target rolls a DC 50 
```

Comment:
We should choose whether to standardize to either concussive or explosive imo

Reply:
I agree completely, and that's not the only problem with this weapon (see outdated reference to door-kicking skill). Unfortunately, the repository reorg interacted oddly with the commits on this branch, which I forked before the reorg, so the entire file is green instead of just the changed bits.

That's a long way of saying that that's outside the scope of this project.

Reply:
Want to make an Issue?

Reply:
Fair enough. I would prefer we templated skill check references like 
"[Skill/Attribute Name] check to [Perform Action]"
instead of "[Perform Action] DC", which in this case would look like "Athletics-STR (or Strength) DC to break down a door"

In this case it looked like a reference to the defunct door-kicking skill that I recall we folded into Athletics.

Reply:
I'll make a note to take care of this in Issue #372